### PR TITLE
Add Support for Other SpikeGLX Interfaces

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR adds support for the SpikeGLXNIDQInterface and SpikeGLXLFPInterface. 

While testing this data, it appeared to not accept the data because of trying to mmap an empty file.

### CODE REVIEW

File: guideGlobalMetadata.json [hunk 1]
```diff
@@ -6,5 +6,7 @@
         "NeuroScopeSortingInterface",
         "BiocamRecordingInterface",
         "IntanRecordingInterface"
+        "SpikeGLXLFPInterface",
+        "SpikeGLXNIDQInterface",
```
Comment (CodyCBakerPhD) [inline guideGlobalMetadata.json:10]:
```suggestion
        "IntanRecordingInterface",
        "SpikeGLXLFPInterface",
        "SpikeGLXNIDQInterface"
```

wrong comma

File: guideGlobalMetadata.json [hunk 2]
```diff
@@ -5,6 +5,8 @@
         "NeuroScopeRecordingInterface",
         "NeuroScopeSortingInterface",
         "BiocamRecordingInterface",
-        "IntanRecordingInterface"
+        "IntanRecordingInterface",
+        "SpikeGLXLFPInterface",
```
Comment (CodyCBakerPhD) [inline guideGlobalMetadata.json:9]:
```suggestion
```

The `SpikeGLXLFPInterface` has been scheduled for deprecation for a while now, so let's not expose it here

The `SpikeGLXRecordingInterface` already accepts the `.lf` band

**Comment by garrettmflynn** [comment]
Updated the ignore list to hide `load_sync_channel` for all interfaces.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Looks great!

Of course, the real use of this interface would be in temporal alignment, but that's a feature for another day

**Pull request merged**